apps/server/src/modules/school/uc/school.uc.spec.ts

```diff
@@ -58,11 +70,26 @@ describe('SchoolUc', () => {
 				expect(schoolService.setMigration).toHaveBeenCalledWith(mockId, true, true, true);
 			});
 		});
+
+		describe('when oauthMigrationFinished is given', () => {
+			it('should call schoolMigrationService to complete the migration when oauthMigrationFinished is true', async () => {
+				await schoolUc.setMigration(mockId, true, true, true, mockId);
```

Comment:
as the "not-nuxt"-client works right now, oauthMigrationFinished = !oauthMigrationPossible. We could think about changing that, because since the last schoolService update we can send these flags now individually or, to honor the clients working, change the first flag here to false ^^

Reply:
It doesnt matter but to test the functionality